[Feature] Build Infographic Screens

yimingc2 opened the pull request:
resolved #112 

## Abstract
This PR will add 6 different infographics screens for PPE ON, Method One, Method Two, Intubation Guide, Intubation Checklist and Extubation to show the PDF.

- [x] `release-notes.txt` has been updated.

## Changes

- Rename `algorithm` to `airway` for the folder and file names
- Rename `intubationAlgorithm` to `extubation`
- Use initial scale for zoomable images which is simpler than before
- Add new routes
- Add 6 new pages for infographics

## Screenshots
https://d.pr/free/v/CD4OYw

## Things to note
There is one thing I'm not sure. Why we don't  put intubation_checklist.png in the infographics folder?

Commit:
Add infographic for ppe

Commit:
Apply infographic page to the sections in airway

Rename Algorithm to airway
Rename intubation algorithm to extubation
Use initial scale for zoomable image

Commit:
Update release note

lukesleeman reviewed (commented):
I ran your code and its looking pretty good!  There are two minor changes I would request

- [x] Remove the padding at the top of the screen once we are zoomed in
- [x] Remove the old "infographic" buttons from the PPE page

# Remove the padding at the top of the screen once we are zoomed in
Once you zoom in on an image there is some padding at the top of the screen
![image](https://user-images.githubusercontent.com/902252/78228471-76be3d80-751a-11ea-9bae-171fba10ae9f.png)

This padding should go away once you zoom

# Remove the old "infographic" buttons from the PPE page
On the PPE page we have two buttons that don't lead anywhere
![image](https://user-images.githubusercontent.com/902252/78228583-9d7c7400-751a-11ea-92b8-eabb2d5bf45f.png)

You can remove them and update the screen to look like this:
![image](https://user-images.githubusercontent.com/902252/78228629-abca9000-751a-11ea-9e17-374ec08fe452.png)

Commit:
Change checklist image location

Remove the top padding of zoomable image in the infographic page
Change the toolbar text style in PPE page

lukesleeman commented on release-notes.txt:
```diff
@@ -1,6 +1,9 @@
 ---- Friday 3rd April, 2020 ----
 * Disclaimer screen on first run of the app, updated, fixes and supports versioning of the disclaimer
 
+---- Thursday 2nd April, 2020 ----
```
Might want to move this to Saturday, as thats when the actual changes will happen in the app on testflight, etc 🙂

lukesleeman commented on lib/widget/reusable_zoomable_widget.dart:
```diff
@@ -5,36 +5,27 @@ class ZoomablePhoto extends StatelessWidget {
-        ),
+              imageProvider: AssetImage(imageUrl),
+              initialScale: PhotoViewComputedScale.contained * 0.9,
+              basePosition: Alignment.topCenter * 0.9,
```
I wanted to tag @workerbee22 on this to get suggestions.  What this is trying to achieve is add some padding at the top of the image, as is shown in the designs:

![image](https://user-images.githubusercontent.com/902252/78417897-af792680-7682-11ea-9149-486aa8d0540f.png)

When I run the code though, because the `0.9` thing is a percentage, it seems to not quite work out for different sized images.

@workerbee22 is there a way to just add 16dp of padding here?

workerbee22 replied:
Yep wrap that widget in a Padding widget and specify 16pt

yimingc2 replied:
@workerbee22 The problem is this third party library `photo_view` does percentage setting rather than points. And the paddings around the image needs to disappear when zoom in.

lukesleeman replied:
I'm happy to leave this for a later issue as its not a major thing.  If @workerbee22 agrees we can get your PR merged and fix this up a low priority design tweak

workerbee22 replied:
Thats OK. Also not sure why we wrap it with `ClipRect` ? That is an expensive operation.

workerbee22 replied:
We'll probably have to look at this separatly.

lukesleeman reviewed (commented):
I've looked through and run it in the android emulator and left a few comments.  Generally it seems good, with one small minor query on the way we set the default image size

sallychung reviewed (approved):
Tested on both iPhons 5s and iPhone 11 Pro Max.

Looks good ✨

workerbee22 commented:
@yimingc2 This branch seems to be way behind master and I get conflicts.

lukesleeman commented:
> @yimingc2 This branch seems to be way behind master and I get conflicts.

This could be something with @workerbee22 's setup - I'm helping him look into it at the moment

yimingc2 commented:
> @yimingc2 This branch seems to be way behind master and I get conflicts.

Hi, can you try deleting your local `feature/build-infographic-screens` and getting it from the origin again? I fixed the conflicts and did a rebase with Master.

lukesleeman commented:
> > @yimingc2 This branch seems to be way behind master and I get conflicts.
> 
> Hi, can you try deleting your local `feature/build-infographic-screens` and getting it from the origin again? I fixed the conflicts and did a rebase with Master.

Thanks 😁. I'm talking to mike and it looks like we are all up and running now 👍

workerbee22 commented on lib/view/ppe/ppe_view.dart:
```diff
@@ -15,7 +15,7 @@ class PPEView extends StatefulWidget {
 
 class _PPEViewState extends State<PPEView> {
-  final title = 'PPE';
+  final title = 'PPE Techniques';
```
Card on Home screen says 'PPE Technique' but the appBar here says plural ie. PPE Techniques', we should be consistent.
```suggestion
  final title = 'PPE Technique';
```

yimingc2 replied:
If I add a top padding then when the user zoom in, the padding will not disappear as we expect.

yimingc2 replied:
In Zeplin, it is `PPE Techniques` in the toolbar. We'd better update the design as well.

workerbee22 replied:
@yimingc2 can you Slack message MArc and let him know?

workerbee22 commented on lib/widget/reusable_zoomable_widget.dart:
```diff
@@ -5,36 +5,27 @@ class ZoomablePhoto extends StatelessWidget {
+      children: <Widget>[
+        Expanded(
+          child: ClipRect(
+            child: PhotoView(
```
```suggestion
            child: PhotoView(
               child: Padding(
                  padding: const EdgeInsets.all(16.0),
```

workerbee22 replied:
Wrap with Padding to better space under AppBar.

Note: missing bracket ')' here but hard to made code changes in Suggestions on GitHub and get the brackets right. So re-check this locally first.

yimingc2 replied:
The paddings around is fixed and will not disappear when the user zooms in. @lukesleeman Actually I prefer not to have a fix padding around the image. If the top position looks weird, how about put the image in the center?

workerbee22 replied:
Oh yeah.Lets tweak this later.

workerbee22 reviewed (approved):
There are some more little things, but can be tweaked later. 

LGTM.

Commit:
Fix PPE toolbar title

Remove ClipRect because it’s not necessary here
Update release note
